entry_types/scrolled/package/src/editor/views/EditSectionView.js

```diff
@@ -8,6 +8,19 @@ export const EditSectionView = EditConfigurationView.extend({
   translationKeyPrefix: 'pageflow_scrolled.editor.edit_section',
 
   configure: function(configurationEditor) {
+    const editMotifAreaMenuItem = {
+      name: 'editMotifArea',
+      label: I18n.t('pageflow_scrolled.editor.edit_section.edit_motif_area'),
```

Comment:
I changed the menu item and header text to talk about "motif areas" instead of "image motifs". I left the prompt to select the most important part of the image unchanged since we only display a poster image. So technically it is not completely wrong after all.

I also added a checklist item to REDMINE-17296 as a reminder to also make the help entry talk about background videos instead of only images.